src/main/java/seedu/address/model/profile/exceptions/SimilarProfileException.java

```diff
@@ -4,8 +4,8 @@
  * Signals that the operation will result in duplicate Profiles (Profiles are duplicates if they have the same
```

Comment:
Perhaps the Javadocs here can be updated to reflect the renaming of the exception, from duplicates to similar.